game/player.py

```diff
@@ -14,7 +16,5 @@ def color(self):
     def is_winner(self, board):
         return board.is_game_over() and board.winner_color() == self._color
 
-    @abstractmethod
     def play(self, board):
-        """This method must be overridden"""
-        pass
+        board.put_chip(self._strategy.return_column(board), self._color)
```

Comment:
Just for readability purposes it may be a good idea to use a temp var here:
```
column_to_play = self._strategy.return_column(board)
board.put_chip(column_to_play, self._color)
```